### PR TITLE
Update katello package dependencies

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 1]
```diff
@@ -79,6 +80,7 @@ BuildRequires: %{?scl_prefix}rubygem(stomp)
 BuildRequires: %{?scl_prefix}rubygem(gettext_i18n_rails)
 BuildRequires: %{?scl_prefix}rubygem(apipie-rails) >= 0.5.14
 BuildRequires: %{?scl_prefix}rubygem(fx) < 1.0
+BuildRequires: %{?scl_prefix}rubygem(pg)
```
Comment (evgeni) [inline packages/katello/rubygem-katello/rubygem-katello.spec:83]: Can you please drop the `BuildRequires: %{?scl_prefix}rubygem(pg)` in line 28? That was added as a workaround until Katello's gemfile was fixed.

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 2]
```diff
@@ -111,26 +113,16 @@ Obsoletes: %{?scl_prefix}rubygem-bastion
...
-BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
-BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 4.2.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 4.6.0
 BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 5.0.0
-BuildRequires: %{?scl_prefix}npm(babel-loader) >= 8.0.0
-BuildRequires: %{?scl_prefix}npm(babel-loader) < 9.0.0
-BuildRequires: %{?scl_prefix}npm(identity-obj-proxy) >= 3.0.0
-BuildRequires: %{?scl_prefix}npm(identity-obj-proxy) < 4.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/find-foreman) >= 4.8.0
```
Comment (evgeni) [inline packages/katello/rubygem-katello/rubygem-katello.spec:118]: what's that?

Comment (ekohl) [reply packages/katello/rubygem-katello/rubygem-katello.spec:118]: The result of https://community.theforeman.org/t/rfc-dont-duplicate-npm-packages-in-katello-and-plugins-for-react-testing/18725. The difference is that with Ruby everything is executed from Foreman. Foreman knows where the plugins are and can chdir if needed. The frontend people like to treat plugins as standalone apps and Foreman as a dependency so they come up with their own solutions.

File: packages/katello/rubygem-katello/rubygem-katello.spec (hunk 1 above)
Comment (ekohl) [reply packages/katello/rubygem-katello/rubygem-katello.spec:82]: Done

File: packages/katello/rubygem-katello/rubygem-katello.spec (hunk 2 above)
Comment (evgeni) [reply packages/katello/rubygem-katello/rubygem-katello.spec:117]: k

**Comment by evgeni** [comment]
```
DEBUG util.py:598:  Error: No Package found for tfm-npm(@theforeman/find-foreman) < 5.0.0
DEBUG util.py:598:  Error: No Package found for tfm-npm(@theforeman/find-foreman) >= 4.8.0
DEBUG util.py:598:  Error: No Package found for tfm-npm(downshift) >= 5.4.2
DEBUG util.py:598:  Error: No Package found for tfm-rubygem(pulp_rpm_client) >= 3.4.2
```

Who is going to package those?

**Comment by ekohl** [comment]
Can I kindly ask you to take a look?

**Comment by tbrisker** [comment]
pulp_rpm_client was packaged in https://github.com/theforeman/foreman-packaging/pull/5559. 
downshift and find-foreman were added by @johnpmitsch, I'm not sure if find-foreman needs packaging or not? if i'm not mistaken it is only used by ui tests.

**Comment by ekohl** [comment]
I'll leave that up to others. If they can be skipped, a PR to https://github.com/theforeman/foreman-packaging/blob/9da7a9b9b0319c3f36f543168341faa2a78b13b4/update-requirements#L10 would be welcome, otherwise a PR to add them.

**Comment by johnpmitsch** [comment]
@ekohl @ekohl Downshift is used in production, it existed previously, but was upgraded. I'll follow the docs to update

find-foreman is just used in testing, I can add it to the exclude list

**Comment by johnpmitsch** [comment]
opened PRs for downshift: https://github.com/theforeman/foreman-packaging/pull/5574 and for find-foreman, which was already merged: https://github.com/theforeman/foreman-packaging/pull/5573

**Comment by tbrisker** [comment]
All needed changes are in, can you please update this PR @ekohl ?

**Comment by ekohl** [comment]
Updated.

**Comment by tbrisker** [comment]
This is odd, both of these packages have been built:
```
DEBUG util.py:598:  Error: No Package found for tfm-npm(downshift) >= 5.4.2
DEBUG util.py:598:  Error: No Package found for tfm-rubygem(pulp_rpm_client) >= 3.4.2
```
Perhaps mashing/tagging is needed?

**Comment by ekohl** [comment]
> Perhaps mashing/tagging is needed?

No, the actual builds need to happen, but https://ci.theforeman.org/job/foreman-packaging-rpm-nightly-release/ is failing to build rubygem-katello before it attempts to build those packages.

**Comment by tbrisker** [comment]
> No, the actual builds need to happen, but https://ci.theforeman.org/job/foreman-packaging-rpm-nightly-release/ is failing to build rubygem-katello before it attempts to build those packages.

Oh, right. But isn't this change needed to get rubygem-katello to build? how do we break the loop?

**Comment by ekohl** [comment]
I think by manually running `obal release`. @theforeman/packaging mind taking a look?

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
closure expected

**Pull request merged**